src/port1.0/port.tcl

```diff
@@ -54,9 +54,9 @@ namespace eval port {
 	proc run_callbacks {} {
 		variable _callback_list
 		foreach callback ${_callback_list} {
-			ui_debug "Running callback ${callback}"
+			ui_debug2 "Running callback ${callback}"
 			${callback}
-			ui_debug "Finished running callback ${callback}"
+			ui_debug2 "Finished running callback ${callback}"
 		}
```

Comment:
This is one suggested change, if we ultimately decide to merge this work. Though there are plenty of other opportunities in our portgroups too.